### PR TITLE
Improve discoverability for installing scripts from templates

### PR DESCRIPTION
**Pull request by jsoref**
Improve Extender descriptive text.

Fix zaproxy/zaproxy#5746

### CODE REVIEW

File: addOns/scripts/src/main/resources/org/zaproxy/zap/extension/scripts/resources/Messages.properties [hunk 1]
```diff
@@ -105,6 +106,8 @@ scripts.script.load.charset.selected.error = Unable to properly read the script
 
 scripts.type.extender					= Extender
 scripts.type.extender.desc				= Extender scripts add new functionality, including graphical elements and new API end points.\n\n\
+To install a script from the library, right click it and choose \n\
```
Comment (thc202) [inline addOns/scripts/src/main/resources/org/zaproxy/zap/extension/scripts/resources/Messages.properties]: - I don't think we should add this here. Why just this type and not all of the others? IMO it should be mentioned in all of them (e.g. inject this message before the type description in all).
 - `library` → `templates`

Comment (kingthorin) [reply addOns/scripts/src/main/resources/org/zaproxy/zap/extension/scripts/resources/Messages.properties]: That's a good idea!

**Comment by thc202** [comment]
@jsoref are you able to address the comment? Should we?

**Comment by jsoref** [comment]
Sorry, I don't think I'll be able to get to this any time soon, I think it's better if someone else does it, thanks.

**Comment by thc202** [comment]
Comment addressed and updated the changelog.

**Pull request merged**